### PR TITLE
Log everything to stdout 

### LINKED ISSUE #63: Add stdout as possible log target
When a process is run as a system daemon by some kind of supervisor, often stdout is captured and observable by the supervisor. This is at least the case for systemd, which also provides the `journalctl` command to view logs. So by making it possible to log to stdout we enable ourselves to have all our logs in the same place.

### PR DESCRIPTION
**Pull request by Rembane**

### CODE REVIEW

File: app/Main.hs [hunk 1]
```diff
@@ -81,22 +78,22 @@ main =
...
                 upd                      <- newMVar () -- putMVar when to update
                 mgr                      <- newTlsManager
-                logHandle <- openFile (view cLog config) AppendMode
                 (viewRef, refreshAction) <- runLoggingT
                   (runReaderT refresh (ClientContext config mgr))
                   print
-                Async.concurrently_ 
-                  (Async.concurrently_ 
+
+                Async.concurrently_
```
Comment (Rembane) [inline app/Main.hs:85]: I added a newline and suddenly the differ goes all excited.

Comment (The1Penguin) [reply app/Main.hs:85]: It had spaces after the underscore before, so that's why it is getting so excited

**Review by The1Penguin — APPROVED** [review]
Looks good, makes more sense to log to stdout when having it as a docker container, which is the use case

File: app/Main.hs (hunk 1 above)
Comment (Rembane) [reply app/Main.hs:85]: That explains it! Thank you.

**Pull request merged**